packages/ui/components/input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { InputHTMLAttributes, ReactNode, forwardRef, useId } from "react";
+import { classNames } from "@documenso/lib";
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  label?: string;
+  labelSrOnly?: boolean;
```

Comment:
Maybe it will be a bit more flexible to have `labelClassName` instead of this prop? WDYT?

Reply:
No I don't think it's a good idea :)